appcelerator/titanium_mobile
# [TIMOB-24732] Fix issue where build process would continue before zip…

… creation had finished

**JIRA:** https://jira.appcelerator.org/browse/TIMOB-24732

Verification:

1. Get the [titanium-google-maps](https://github.com/hansemannn/titanium-google-maps) module
2. Build the module using `ti build -p ios -l trace`
    - Build process should not error out, and the example app should launch on device
